src/fetchData.js

```diff
@@ -1,41 +1,41 @@
-function fetchData() {
-  let userData =  fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData")
-    .then(response => response.json())
-    .then(data => {
-      return data.userData;
-    })
-    .catch(err => console.log(err.message))
+// function fetchData() {
+//   let userData =  fetch("https://fe-apps.herokuapp.com/api/v1/fitlit/1908/users/userData")
+//     .then(response => response.json())
+//     .then(data => {
+//       return data.userData;
+//     })
+//     .catch(err => console.log(err.message))
 
```

Comment:
I think we should leave this file separate.

Reply:
We can definitely move it back to a separate file. We ran into issues with the chain breaking and not getting the data we wanted so we moved it onto the scripts file to debug and get a better understanding of the logic. It's much cleaner to have it on a separate file though.